frontend/src/components/jobs/result/visualizer/VisualizerTooltip.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { CopyToClipboardButton } from "@certego/certego-ui";
+import { Button, UncontrolledTooltip } from "reactstrap";
+import { MdContentCopy } from "react-icons/md";
+import { AiOutlineLink } from "react-icons/ai";
+
+export function VisualizerTooltip({
+  idElement,
+  copyText,
+  link,
+  description,
+  disable,
+}) {
+  return (
+    <UncontrolledTooltip target={idElement} placement="right" autohide={false}>
+      <div className="p-0 my-2 d-flex justify-content-start">
+        <Button className="mx-1 p-2" size="sm">
+          <CopyToClipboardButton id={idElement} text={copyText}>
+            <MdContentCopy /> Copy
+          </CopyToClipboardButton>
+        </Button>
+        <Button className="mx-1 p-2" size="sm" disabled={disable || !link}>
+          <AiOutlineLink />
+          <a
+            href={link}
+            target="_blank"
+            rel="noreferrer"
+            style={{ textDecoration: "none" }}
+          >
+            &nbsp;Link
+          </a>
+        </Button>
+      </div>
```

Comment:
For both copy button and link button the actions (copy and link) work only if the user clicks the labels "copy" and "text". In case the analyst clicks the button, but not the label it doesn't work.